repositories.json

```diff
@@ -103,5 +103,6 @@
   "TheSICKnastySuite": "https://raw.githubusercontent.com/KCisSICKnasty/sicknasty-suite/main/modlist.json",
   "Scrolls_of_Schtevie": "https://raw.githubusercontent.com/HerrSchtevie/Scrolls-of-Schtevie/main/modlists.json",
   "Kahnezzer": "https://raw.githubusercontent.com/Kahnezzer/OldNewEngland/main/modlist.json",
-  "CerberusX1": "https://raw.githubusercontent.com/CerberusX1/modlists/refs/heads/main/modlists.json"
+  "CerberusX1": "https://raw.githubusercontent.com/CerberusX1/modlists/refs/heads/main/modlists.json",
+  "LennysVR84": https://raw.githubusercontent.com/LennysVR84/Nordic-Adventures---The-Crew-Lite/refs/heads/main/modlists.json
```

Comment:
Please remove the "/refs/heads" part of your url.
Compare with the other documented repositories how your url should look like.